docs/contributing.rst

```diff
@@ -71,12 +71,43 @@ or functions.
 Testing
 ^^^^^^^
 
-Twine is tested against Python versions 3.6, 3.7, and 3.8. To run these tests
-locally, you will need these versions of Python installed on your machine.
+We use `pytest`_ for writing and running tests.
+
+To run the tests in your virtual environment, run:
+
+.. code-block:: console
+
+  tox -e py
+
+To pass options to ``pytest``, e.g. the name of a test, run:
+
+.. code-block:: console
+
+  tox -e py -- tests/test_upload.py::test_exception_for_http_status
+
+You can also set the ``PYTEST_ADDOPTS`` environment variable to use the same
+options on every test run. For example, to skip integration tests and turn
+off the coverage report, run:
+
+.. code-block:: console
+
+  export PYTEST_ADDOPTS=-k "not integration" --cov-report=
+
+Twine is continuously tested against Python 3.6, 3.7, and 3.8 using `Travis`_.
+To run the tests against a specific version, e.g. Python 3.6, you will need it
+installed on your machine. Then, run:
```

Comment:
I have python3.6 installed locally, but then I ran this command in my virtualenv which has python3.8, but it failed.

```
(.env) $ python --version
Python 3.8.2
(.env) DeveshSinghMac:twine devesh$ tox -e py36
GLOB sdist-make: /Users/devesh/twine/setup.py
py36 create: /Users/devesh/twine/.tox/py36
ERROR: InterpreterNotFound: python3.6
_________________________________________________________________ summary _________________________________________________________________
ERROR:  py36: InterpreterNotFound: python3.6
```

Did I miss something here?

Reply:
Why are you running tox from a virtualenv?